### DISCUSSION TITLE
How can I set up this plugin so that it will not be automatically loaded when there are no args

### DISCUSSION THREAD

**Discussion by JuanZoran** [rmagatti/auto-session#188] [Q&A]
as I said that, I have set dashboard for neovim

- what I want to setup is that, I can open the dashboard when there is no arguments
- I can set a keyword in dashboard for open my session

so how can i do that ? 
sorry about my poor English

**Comment by JuanZoran** [comment]
emm... I found what can I do for that...

**Reply by rmagatti** [reply]
To do that you'll most likely have to disable auto-session by default. Auto Session has a predefined behaviour when you open Neovim with no args which is to use the current working directory to determine what session it should load and restore that. You can disable automatic session restoring to avoid that.

**Comment by JuanZoran** [comment]
Thank you for your reply! :)
